returnn/training.py

```diff
@@ -159,13 +180,21 @@ def __init__(
             for k in stored_epochs
             if k in self.keep_epochs
         }
-        if self.returnn_config.get("use_tensorflow", False):
+        if self.returnn_config.get("use_tensorflow", False) or self.returnn_config.get("backend") == "tensorflow":
             self.out_checkpoints = {
                 k: Checkpoint(index_path)
                 for k in stored_epochs
                 if k in self.keep_epochs
                 for index_path in [self.output_path("models/epoch.%.3d.index" % k)]
             }
+        if self.returnn_config.get("backend", None) == "torch":
+            self.out_checkpoints = {
+                k: PtCheckpoint(index_path)
+                for k in stored_epochs
+                if k in self.keep_epochs
+                for index_path in [self.output_path("models/epoch.%.3d.pt" % k)]
```

Comment:
We might want to go for 4 digits here...

Reply:
Or infer from `num_epochs` with minimum 3 to not break old paths.